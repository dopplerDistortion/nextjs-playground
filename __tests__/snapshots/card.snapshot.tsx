import {render} from '@testing-library/react';
import {
  Card,
  CardContent,
  CardDescription,
  CardFooter,
  CardHeader,
  CardTitle,
} from '@/components/ui/card';

const DummyCard = () => (
  <Card>
    <CardHeader>
      <CardTitle>Card Title</CardTitle>
      <CardDescription>Card Description</CardDescription>
    </CardHeader>
    <CardContent>
      <p>Card Content</p>
    </CardContent>
    <CardFooter>
      <p>Card Footer</p>
    </CardFooter>
  </Card>
);

it('renders card unchanged', () => {
  const {container} = render(<DummyCard />);
  expect(container).toMatchSnapshot();
});
