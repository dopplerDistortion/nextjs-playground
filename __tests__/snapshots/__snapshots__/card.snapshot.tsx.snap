// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders card unchanged 1`] = `
<div>
  <div
    class="rounded-lg border bg-card text-card-foreground shadow-sm"
  >
    <div
      class="flex flex-col space-y-1.5 p-6"
    >
      <h3
        class="text-2xl font-semibold leading-none tracking-tight"
      >
        Card Title
      </h3>
      <p
        class="text-sm text-muted-foreground"
      >
        Card Description
      </p>
    </div>
    <div
      class="p-6 pt-0"
    >
      <p>
        Card Content
      </p>
    </div>
    <div
      class="flex items-center p-6 pt-0"
    >
      <p>
        Card Footer
      </p>
    </div>
  </div>
</div>
`;
